sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.0.0-beta.4 (Unreleased)
+
+
 ## 1.0.0-beta.3 (2021-01-20)
 
```

Comment:
Was this library released without updating its Changelog?

//cc @jialindai

Reply:
@vcolin7 . Thank you for your check. I updated the changelog.